spec/identity_spec.rb

```diff
@@ -16,24 +19,59 @@
     )
   end
 
-  before do
-    FileUtils.remove_dir(tmp_dir) if File.exists?(tmp_dir)
-    FileUtils.mkdir_p(tmp_dir)
-    File.write(File.join(tmp_dir, 'config'), identity_string)
+  before(:all) do
```

Comment:
what happened here

Reply:
the previous version of this worked great so we need to figure out why thats broken now, this is too complex to be maintainable

Reply:
Previous version wasn't using IPFS, just checking file paths.  If we wanna assume a running daemon that's fine, but then we shouldn't be making our own config dir.  I've been thinking integration not unit.